pages/content/amp-dev/documentation/guides-and-tutorials/integrate/embed-stories.md

```diff
@@ -0,0 +1,198 @@
+---
+$title: Embed stories in web pages
+$order: 3
+description: 'The AMP Story Player enables you to embed a story that users are able to tap or click through, inside of a web page. Follow this step-by-step guide to learn how.'
+formats:
+  - websites
+  - stories 
+---
+
+Stories are a full-screen immersive experience. They’re hosted on the open web with their own URL, making them easily shareable. But, what if you want to integrate stories into your own site, for example, within a blog, product description or news article? 
+
+The [AMP Story Player](https://github.com/ampproject/amphtml/blob/master/spec/amp-story-player.md) enables you to embed a story that users are able to tap or click through, inside of a web page. Follow this step-by-step guide to learn how.
+
+# Display stories in a non-AMP page
+
+You can embed AMP stories inside a non-AMP page, allowing users to tap or click through the experience without leaving the host document!
+
+[example preview="top-frame" playground="false"]
+```html
+<!doctype html>
+    <head>
+    <script async src="https://cdn.ampproject.org/v0.js"></script>
+    <script
+        async
+        src="https://cdn.ampproject.org/amp-story-player-v0.js"
+    ></script>
+    <link
+        href="https://cdn.ampproject.org/amp-story-player-v0.css"
+        rel="stylesheet"
+        type="text/css"
+    />
+    <style>
+        header {
+        height: 8vh;
+        width: 100vw;
+        display: flex;
```

Comment:
What is this good for? Wouldn't `text-align: center` do the same?

Reply:
I've updated the CSS - but heres some ss of how it looks with flex vs center
<img width="1310" alt="Screen Shot 2020-03-19 at 12 08 14 PM" src="https://user-images.githubusercontent.com/23297417/77105156-6bc3c300-69da-11ea-9a46-783665cf12cb.png">
<img width="1353" alt="Screen Shot 2020-03-19 at 12 08 25 PM" src="https://user-images.githubusercontent.com/23297417/77105171-71b9a400-69da-11ea-9fc4-31611216b3ba.png">

Reply:
True, flex is better than. In that case center vertically and horizontally via flex:

```
header {
          height: 8vh;
          color: #545454;
          background-color: #DDB556;
          display: flex;
          align-items: center;
          justify-content: center;
}
```